.gitignore

```diff
@@ -48,6 +48,7 @@ quantum/version.h
 .idea/
 .project
 .settings/
+*.sln
```

Comment:
```suggestion
```

Reply:
Hey! I'm happy to make this change but wanted to ask as a follow up whether you'd want me to commit the solution file that Visual Studio and VS Code generate when using the .NET extensions. 

I figured you wouldn't want that which motivated this change, but since you want it removed I think it would be good to double check my assumption.

Reply:
This change should be made in a separate PR, if you feel it is necessary.

Reply:
Understood, happy to do so.